app/scripts/modules/Schedule.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+var React = require('react');
+
+var Schedule = React.createClass({
+  getInitialState: function() {
+    return {
+      conferences:[]
+    }
+  },
+  componentDidMount: function() {
+    var req = new XMLHttpRequest();
```

Comment:
create an abstraction 'client' or use some instead of bootstraping XMLHttpRequest all the time